temporian/core/operators/arithmetic.py

```diff
@@ -21,21 +23,38 @@
 from temporian.proto import core_pb2 as pb
 
 
-from enum import Enum
+class ArithmeticOperation(str, Enum):
+    ADDITION = "ADDITION"
+    SUBTRACTION = "SUBTRACTION"
+    MULTIPLICATION = "MULTIPLICATION"
+    DIVISION = "DIVISION"
+
+    @staticmethod
+    def prefix(operation: "ArithmeticOperation") -> str:
+        if operation == ArithmeticOperation.ADDITION:
+            return "sum"
+        if operation == ArithmeticOperation.SUBTRACTION:
+            return "diff"
+        if operation == ArithmeticOperation.MULTIPLICATION:
+            return "prod"
+        if operation == ArithmeticOperation.DIVISION:
+            return "div"
+        raise ValueError(f"Unknown operation: {operation}.")
 
 
 class Resolution(str, Enum):
     PER_FEATURE_IDX = "PER_FEATURE_IDX"
     PER_FEATURE_NAME = "PER_FEATURE_NAME"
 
 
-class SumOperator(Operator):
-    """Sum operator."""
+class ArithmeticOperator(Operator):
+    """Arithmetic operator."""
 
     def __init__(
         self,
         event_1: Event,
         event_2: Event,
+        operation: ArithmeticOperation,
         resolution: Resolution = Resolution.PER_FEATURE_IDX,
```

Comment:
No need to double define the default values. Currently, the default values should be only defined in the API.